### DISCUSSION TITLE
--cache-to and --cache-from is not being recognized

### DISCUSSION THREAD

**Discussion by sjm-ho** [containers/buildah#5653] [General]
So I am trying to build an docker image in github actions using the following command:

```
              sudo buildah bud \
              --build-arg APP_ENV = ${{ matrix.env }}
              --layers \
              --cache-to $ECR_REPOSITORY/cache \
              --cache-from $ECR_REPOSITORY/cache \
              -f Dockerfile \
              -t ${{ steps.docker_meta.outputs.tags }} .
```

As you can see I am trying to cache the intermediate layers in a folder named as cache in my $ECR_REPOSITORY
 
 1. But it is not able to build the image, the workflow exits with an error:
 `unknown flag: --cache-to`
 
 2. If I were to cache it in github actions cache how would I do it? ( I am using github. ARC runners not github managed runners )
 
 My buildah version:
 buildah version 1.23.1 (image-spec 1.0.1, runtime-spec 1.0.2-dev)

**Comment by nalind** [comment]
`--cache-from` and `--cache-to` were not introduced until 1.27.

**Comment by sjm-ho** [comment]
Right @nalind 
But this is my docker file

```
FROM ghcr.io/actions/actions-runner:2.317.0

RUN sudo apt-get update \
    && sudo apt-get install -y \
        buildah \
        awscli \
        npm \
        curl \
    && sudo rm -rf /var/lib/apt/lists/*

# Install n package manager
RUN sudo npm install -g n

# Upgrade Node.js to the latest version
RUN sudo n latest

# Update npm to the latest version
RUN sudo npm install -g npm@latest


 # Install Yarn
RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - \
    && echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list \
    && sudo apt-get update \
    && sudo apt-get install -y yarn \
    && sudo rm -rf /var/lib/apt/lists/*

RUN sudo rm -rf /etc/containers/registries.conf
COPY ./dockerfiles/arc-runner/registries.conf /etc/containers/

# Cleanup
RUN sudo apt-get clean \
    && sudo rm -rf /tmp/* /var/tmp/*
```

I am not able to pull or install the latest version of buildah.

So if I do an apt-get install buildah, it is supposed to search the ubuntu repositories which is https://packages.ubuntu.com/jammy/buildah

but here I can't find the versions > 1.27
